renovate.json

```diff
@@ -30,6 +30,48 @@
         "gomod"
       ]
     },
+    {
+      "description": "Group all Kubernetes updates",
+      "enabled": true,
+      "matchPackageNames": [
+        "k8s.io/api",
```

Comment:
I don't understand how both 0.X.Y and 1.X.Y work, but if it gets the job done it's welcome

Reply:
Alright. I've put more wort into this, and now I'm quite confident that it works. Can you please take another look @juanluisvaladas?